src/test/java/com/redhat/darcy/webdriver/CookieTest.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.darcy.webdriver;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.redhat.darcy.web.Cookie;
+import com.redhat.darcy.web.SimpleUrlView;
+import com.redhat.darcy.web.api.Browser;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public class CookieTest {
+
+    @Test
+    public void shouldBeACookie() {
+        new Cookie("Fish", "cod", "", "", null, false);
+    }
+
+    @Test
+    public void shouldValidateAValidCookie() {
+        new Cookie("Fish", "cod", "", "", null, false)
+                .validate();
+    }
+
+    @Test
+    public void shouldFailValidationForACookieWithAnIllegalName() {
+        new Cookie("Fish;", "cod", "", "", null, false)
+                .validate();
+    }
+
+    @Test
+    public void shouldPrintCookieToString() {
+        System.out.println(new Cookie("Fish;", "cod", "", "", LocalDateTime.now(), false)
+                .toString());
+    }
+
+    @Test
+    public void shouldGetCookiesFromGoogle() {
```

Comment:
Actually I found this to be not true.  because it calls `manage()` on the mocked TargetedWebDriver that returns a null for the inner class Options.  I tried using mockito's when and verify but I'm not sure if this can be tested with mocks. I welcome suggestions.